### ISSUE TITLE
"The Great Ace Attorney" Expansion files (OBBs) not detected. Your "The Great Ace Attorney" installation is corrupted.

### ISSUE THREAD

**Issue by LunaticMaverick** [ScarletStudy/DGS1-Android-Release#84]
### About your device

**Device Manufacturer+Model or Emulator+Version?**

**Android Version?** (Kit Kat/Lollipop/Marshmallow/Nougat/Oreo/Pie or the number as in 7.1.0)

**Are you using a Custom ROM?** (If you don't know what this means, then most likely no)

**Is your device rooted?** (If you don't know what this means, then most likely no)

### Your issue

**Please describe the issue you are experiencing:**

**Comment by wabberz** [comment]
#78
#79
#82
#83
https://scarletstudy.gq/2020/03/17/playstore-updates/

**Comment by LunaticMaverick** [comment]
So should i just wait for it to be fixed?

**Comment by wabberz** [comment]
Yep, also look for similar issues first before opening a new one if at all possible.
